gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/root/service/user/impl/UserServiceImpl.java

```diff
@@ -608,12 +607,10 @@ public Single<User> updateFactor(String userId, EnrolledFactor enrolledFactor, i
 
     @Override
     public Completable setMfaEnrollmentSkippedTime(Client client, User user) {
-        EnrollSettings mfaEnrollSettings = getMfaEnrollmentSettings(client);
-        final Boolean active = Optional.ofNullable(mfaEnrollSettings.getForceEnrollment()).orElse(false);
-        if (FALSE.equals(active) && nonNull(user)) {
+        if (nonNull(user)) {
```

Comment:
**question**: is it safe to remove the test on the "forceErollment" boolean?

Reply:
right, I added a check and tests for the Enroll endpoint for it